tests/integration/services.test.js

```diff
@@ -154,123 +155,108 @@ describe('GET /services', () => {
 
 describe('PUT /services/:id', () => {
   let authToken = null;
-  beforeEach(populateDB);
-  beforeEach(async () => {
-    authToken = await getToken('client_admin');
-  });
-  it('should update service', async () => {
-    const payload = { defaultUnitAmount: 15, name: 'Service bis', startDate: '2019-01-16 17:58:15.519', vat: 12 };
-    const response = await app.inject({
-      method: 'PUT',
-      url: `/services/${servicesList[0]._id.toHexString()}`,
-      headers: { Cookie: `alenvi_token=${authToken}` },
-      payload,
+  describe('CLIENT_ADMIN', () => {
+    beforeEach(populateDB);
+    beforeEach(async () => {
+      authToken = await getToken('client_admin');
```

Comment:
Tu peux sauter une ligne entre le `beforeEach` et le `it`

Reply:
done